fetch.bs

```diff
@@ -2049,6 +2052,34 @@ clearly stipulates that <a>connections</a> are keyed on
 <!-- See https://github.com/whatwg/fetch/issues/114#issuecomment-143500095 for when we make
      WebSocket saner -->
 
+<h3 id=http-cache-partitions>HTTP cache partitions</h3>
+
+<p>For <dfn export>determining the HTTP cache partition</dfn>, given <var>request</var>,
+run these steps:
+
+<ol>
+ <li>
+  <p>Let <var>environmentSettings</var> be <var>request</var>'s <a for=request>client</a>, if
+  present, otherwise return null.</p></li>
```

Comment:
This is a bit confusing. I would just have a second step that says

1. If _environmentSettings_ is null, then return null.

Reply:
done